### ISSUE TITLE
Docker image for proj.4

### ISSUE THREAD

**Issue by vaibhavsood** [OSGeo/PROJ#587]
Hi,

Want to check if there is a docker image/Dockerfile for proj.4 which is officially supported? I could not find one under dockerhub official images https://hub.docker.com/explore/

If not, want to check if there is any plan to add an official proj.4 image to dockerhub? https://docs.docker.com/docker-hub/official_repos/#how-do-i-create-a-new-official-repository

**Comment by kbevers** [comment]
Docker is mentioned on the download section on proj4.org, although there is no direct link. Here it is:

https://hub.docker.com/r/osgeo/proj.4/

**Comment by vaibhavsood** [comment]
Thanks for the response!

Reason i asked was i wanted to check if it makes sense to have proj.4 as a docker official image (steps for doing the same are in the link posted in my previous comment, it also mentions the considerations for making an image official)

If that happens, the image comes under Docker's "official" image program, they then take over the daily builds etc, also it gets published under https://hub.docker.com/explore/ - however the maintainence is to be done by upstream

So in summary wanted to check if doing this make sense or hosting the image under https://hub.docker.com/r/osgeo/proj.4/ would be the only way for this docker image?

**Comment by kbevers** [comment]
Ping @hobu - this is your area of expertise.

**Comment by hobu** [comment]
> check if it makes sense to have proj.4 as a docker official image

proj.4 is a building block library, and while it has a couple of command line utilities, 90% of the time, it is used in context with other software packages. I don't think proj4 should become an "official" image because it would be yet another thing to support for limited gain.
